lms/djangoapps/courseware/models.py

```diff
@@ -248,3 +255,113 @@ class Meta(object):   # pylint: disable=missing-docstring
 
     field = models.CharField(max_length=255)
     value = models.TextField(default='null')
+
+
+SCORE_CHANGED = Signal(
+    providing_args=[
+        'points_possible',  # Maximum score available for the exercise
+        'points_earned',   # Score obtained by the user
+        'user',  # User object representing the student
+        'course',  # CourseLocator object representing the course
+        'usage'  # BlockUsageLocator indicating the courseware instance
```

Comment:
You'll probably want to pass string representations of these instead of the locator objects themselves, because locators won't serialize to JSON (which is what celery is going to do to it when it pushes stuff into rabbitmq).